cmd/habitat/community/consensus/cluster/cluster.go

```diff
@@ -61,36 +59,23 @@ func (cm *ClusterManager) Start(proxyRules *proxy.RuleSet) error {
 		return err
 	}
 
-	// Restart any existing communities
-	comDirs, err := ioutil.ReadDir(compass.CommunitiesPath())
-	if err == nil {
-		for _, dir := range comDirs {
-			err := cm.RestoreNode(dir.Name())
-			if err != nil {
-				log.Error().Err(err).Msgf("error restoring cluster for community %s", dir.Name())
-			}
-		}
-	} else if !errors.Is(err, os.ErrNotExist) {
-		return err
-	}
-
 	return nil
 }
 
-func (cm *ClusterManager) CreateCluster(communityID string) error {
-	return cm.raftClusterService.CreateCluster(communityID)
+func (cm *ClusterManager) CreateCluster(communityID string, initState []byte) (<-chan state.StateUpdate, error) {
```

Comment:
these functions just call the underlying raftClusterService functions - why can't we just call these directly? are you planning to add more here at some point?

Reply:
yeah one extra layer of abstraction in case we want to switch away from Raft at some point. The main reason why we might want to do this is that for communities with dozens of nodes, Raft might be too expensive in terms of network usage and latency. But we could also figure out a system where only a subset of the nodes in the community participate in the Raft network